pkgs/by-name/lc/lcov/package.nix

```diff
@@ -44,12 +44,11 @@ stdenv.mkDerivation rec {
 
   preBuild = ''
     patchShebangs --build bin/{fix.pl,get_version.sh} tests/*/*
-    patchShebangs --host bin/{gen*,lcov,perl2lcov}
-    makeFlagsArray=(PREFIX=$out LCOV_PERL_PATH=${lib.getExe perl})
+    makeFlagsArray=(PREFIX=$out)
   '';
 
   postInstall = ''
-    for f in "$out"/bin/{gen*,lcov,perl2lcov}; do
+    for f in "$out"/bin/{gen*,lcov,llvm2lcov,perl2lcov}; do
```

Comment:
I think of "cleanup" as mostly non-functional, but adding llvm2lcov here seems like a packaging fix? Can you explain what's happening?

Nit: The commit message body should consist of full sentences; start with capital letter, end with period.